scalr/assets/service_checks.json

```
@@ -0,0 +1,16 @@
+[
+    {
+        "agent_version": "7.39.2",
```

```suggestion
        "agent_version": "7.40.1",
```

This is the latest version of the agent